tools/atip/atip/common/steps.py

```diff
@@ -59,6 +59,15 @@ def switch_to_app_name(context, app_name):
         assert False
 
 
+@step(u'quit "{app_name}"')
```

Comment:
Don't put this method in comon folder since we are integrating uiautomator for android, and this method is specific for webdriver.